pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -622,6 +622,19 @@ const outputLabelConfJsonParseNoretryTemplate = `{{- define "outputLabelConfJson
 	merge_json_log false
 	replace_json_log true
   </filter>
+{{ if .Target.Syslog.AddLogSource }}
+  <filter **>
+	@type record_modifier
+	<record>
+	  namespace_info      ${if record.has_key?('kubernetes'); "namespace_name=" + record['kubernetes']['namespace_name']; else nil; end}
+	  pod_info            ${if record.has_key?('kubernetes'); "pod_name=" + record['kubernetes']['pod_name']; else nil; end}
+	  container_info      ${if record.has_key?('kubernetes'); "container_name=" + record['kubernetes']['container_name']; else nil; end}
+	  message_info        ${if record['message'].is_a?(Hash); require 'json'; "message=#{record['message'].to_json}" ; else "message=" + record['message']; end}
+	  message             ${if record.has_key?('kubernetes'); record['namespace_info'] + "," + record['pod_info'] + "," + record['container_info'] + "," + record['message_info']; else record['message']; end}
```

Comment:
In the legacy way, the format was "namespace_name=xxx, container_name=xxx, pod_name=xxx, message=xxx".
https://github.com/openshift/origin-aggregated-logging/blob/ee4935b6e1a4507a3714452e99acf0c945c2568c/fluentd/out_syslog.rb#L138

Could you please change the format to the following to keep the format compatibility(a space is needed after ",", the order is namespace=>container=>pod) between the new way and the legacy way:
```
message             ${if record.has_key?('kubernetes');  record['namespace_info'] + ", " + record['container_info'] + ", " + record['pod_info'] + ", "  + record['message_info']; else record['message']; end}
```

Reply:
Thank you for your work.

> Could you please change the format to the following to keep the format compatibility(a space is needed after ",", the order is namespace=>container=>pod) between the new way and the legacy way:
> 
> ```
> message             ${if record.has_key?('kubernetes');  record['namespace_info'] + ", " + record['container_info'] + ", " + record['pod_info'] + ", "  + record['message_info']; else record['message']; end}
> ```

Is this change acceptable for you? If not, is there any reason?